ballerina/utils.bal

```diff
@@ -94,3 +95,18 @@ public isolated function filterRecord(record {} 'object, string[] fields) return
     }
     return retrieved;
 }
+
+public isolated function getSheetIds(sheets:Client googleSheetClient, record {|SheetMetadata...;|} metadata, string spreadsheetId) returns map<int>|Error {
+    map<int> sheetIds = {};
+    sheets:Sheet|error sheet;
+    foreach string key in metadata.keys() {
+        sheet = googleSheetClient->getSheetByName(spreadsheetId, metadata.get(key).tableName);
+        if sheet is error {
+            return <Error>error(string `Error: sheet cannot be found : ${metadata.get(key).tableName}`);
+        }
+        sheetIds[key] = sheet.properties.sheetId;
+    }
+    return sheetIds;
+    
+    
+}
```

Comment:
```suggestion
}

```

Reply:
updated in [4346c39](https://github.com/ballerina-platform/module-ballerina-persist/pull/201/commits/4346c3959a465aeb05de0872b94ea1056a715a9a)